src/main/java/org/isf/dicom/gui/DicomGui.java

```diff
@@ -515,7 +514,6 @@ public void detail() {
 	 * @param args
 	 */
 	public static void main(String[] args) {
-		PropertyConfigurator.configure(new File("./rsc/log4j.properties").getAbsolutePath());
 		ApplicationContext context = new ClassPathXmlApplicationContext("applicationContext.xml");
```

Comment:
here is a hard coded file name, is this fine ?

Reply:
you mean the removed line?

Reply:
No, I meant the line that is going to stay in the code, and any other line hardcoding file names, since that is the goal or the PR :)  

 517  ->  `ApplicationContext context = new ClassPathXmlApplicationContext("applicationContext.xml");`

Reply:
yeah, filename is ok, the issue is about hardcoded "paths".
Anyway, in this PR we are only caring about configuration files, other paths that could be resolved are the ones for icons and images.